content/docs/glossary/common-rkvst-terms/index.md

```diff
@@ -0,0 +1,37 @@
+| [ABAC](https://docs.rkvst.com/docs/rkvst-basics/managing-access-to-an-asset-with-abac/)                | Attribute-Based Access Control; policy that allows you to grant fine-grain access to members of your tenancy         |
+| [access policy](https://docs.rkvst.com/docs/overview/core-concepts/#access-policies)       | grants chosen asset and event access to stakeholders                                                                 |
+| [app registration](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/#creating-an-app-registration)    | client ID and client secret credentials that are used to access the RKVST API                                        |
+| [asset](https://docs.rkvst.com/docs/overview/core-concepts/#assets)               | an RKVST asset has a collection of attributes that describes its current state and a complete life history of events |
+| [asset_attributes](https://docs.rkvst.com/docs/rkvst-basics/creating-an-asset/#creating-an-asset)    | key-value pairs that represent information about an asset                                                            |
+| [bearer token](https://docs.rkvst.com/docs/rkvst-basics/creating-an-asset/#creating-an-asset)        | access token for RKVST API; created using app registration credentials                                               |
+| [behaviours](https://docs.rkvst.com/docs/rkvst-basics/creating-an-asset/#creating-an-asset)          | detail what class of events in an asset lifecycle you might wish to record                                           |
+| [compliance policy](https://docs.rkvst.com/docs/beyond-the-basics/compliance-policies/)   | user-defined rule sets that assets can be tested against                                                             |
+| [event](https://docs.rkvst.com/docs/overview/core-concepts/#events)               | tracks key moments of an asset lifecycle; details of Who Did What When to an Asset                                   |
+| [event_attributes](https://docs.rkvst.com/docs/rkvst-basics/creating-an-event-against-an-asset/#creating-events)    | key-value pairs that represent information about an event                                                            |
+| [OBAC](https://docs.rkvst.com/docs/rkvst-basics/sharing-assets-with-obac/)                | Organization-Based Access Control; policy allows sharing with the root user of another organization                  |
+| [operation](https://docs.rkvst.com/docs/rkvst-basics/creating-an-event-against-an-asset/#creating-events)           | class of event being performed                                                                                       |
+| [public asset](https://docs.rkvst.com/docs/beyond-the-basics/public-attestation/)        | assets that can be used to publicly assert data, accessible by URL                                                   |
```

Comment:
I'd like this to show that someone can read a publishers public asset easily.

"a publicly available asset, which can be used to assert _"something"_, accessible by URL" Is "data" the right word? (it might be)

Reply:
"data" is the word currently used in the public attestation section, should we change in both places? "something" might be better and more broad